src/Microsoft.DotNet.Build.Tasks/Targets/partialfacades.targets

```diff
@@ -52,13 +52,27 @@
       $(TargetsTriggeredByCompilation);FindPartialFacadeContractReference;FillPartialFacade
     </TargetsTriggeredByCompilation>
   </PropertyGroup>
+  
+  <!-- If a reference assembly project exists for the partial facade, prefer that over the package  -->
+  <Target Name="FindPartialFacadeProjectReference" Condition="'$(IsPartialFacadeAssembly)' == 'true'" BeforeTargets="AssignProjectConfiguration">
+    <PropertyGroup>
+      <_referenceAssemblyProjectRelativeToRoot>$([System.IO.Path]::Combine('ref', '$(APIVersion)', '$(AssemblyName).csproj'))</_referenceAssemblyProjectRelativeToRoot>
+      <_referenceAssemblyProject>$([System.IO.Path]::Combine($([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), $(_referenceAssemblyProjectRelativeToRoot))), $(_referenceAssemblyProjectRelativeToRoot)))</_referenceAssemblyProject>
+    </PropertyGroup>
+    <ItemGroup>
+      <ProjectReference Include="$(_referenceAssemblyProject)" Condition="'$(_referenceAssemblyProject)' != ''">
```

Comment:
GetDirectoryNameOfFileAbove does an exists check but I guess I do need that since the path combine will give me a bogus path in the case GetDirectoryNameOfFileAbove fails.